addons/l10n_bh/data/tax_report_full.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo auto_sequence="1">
+    <record id="l10n_bh_tax_report_full" model="account.report">
+        <field name="name">Full VAT Return</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.bh"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="l10n_bh_tax_report_full_base" model="account.report.column">
+                <field name="name">Base</field>
+                <field name="expression_label">base</field>
+            </record>
+            <record id="l10n_bh_tax_report_full_adjustment" model="account.report.column">
+                <field name="name">Adjustment/Apportionment</field>
```

Comment:
The name seems too long. The report looks a bit weird in the interface with the huge column